src/main/java/com/depromeet/todo/infrastructure/util/SnowFlakeIdGenerator.java

```diff
@@ -0,0 +1,23 @@
+package com.depromeet.todo.infrastructure.util;
+
+import com.depromeet.todo.domain.IdGenerator;
+import org.springframework.stereotype.Component;
+
+import java.util.Random;
+
+@Component
+public class SnowFlakeIdGenerator implements IdGenerator {
+    private static final Random RANDOM = new Random();
```

Comment:
https://stackoverflow.com/questions/23396033/random-over-threadlocalrandom

해당 아티클에 대해 어떻게 생각하시나요?

Reply:
Random 클래스가 thread-safe 하고, multi-thread 환경에서 경합이 발생할 수 있다는 내용은 처음 알았네요. 알려주셔서 감사합니다 :)
ThreadLocalRandom 으로 변경하겠습니다

#### Random
... Instances of java.util.Random are threadsafe. However, the concurrent use of the same java.util.Random instance across threads may encounter contention and consequent poor performance. Consider instead using ThreadLocalRandom in multithreaded designs. ...
(https://docs.oracle.com/javase/8/docs/api/java/util/Random.html)


#### ThreadLocalRandom
... When applicable, use of ThreadLocalRandom rather than shared Random objects in concurrent programs will typically encounter much less overhead and contention. Use of ThreadLocalRandom is particularly appropriate when multiple tasks (for example, each a ForkJoinTask) use random numbers in parallel in thread pools. ...
(https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/ThreadLocalRandom.html)